Makefile

```diff
@@ -18,11 +18,21 @@ CPPCHECK=ls lib/*.cc khmer/_khmermodule.cc | grep -v test | cppcheck -DNDEBUG \
 	 --quiet -Ilib -Ithird-party/bzip2 -Ithird-party/zlib \
 	 -Ithird-party/smhasher
 
-all: khmer/_khmermodule.so
 
+## all                  : default task; compile C++ code, build shared object library
+all: sharedobj
+
+## help                 : print this help message and exit
+help: Makefile
+	@sed -n 's/^##//p' $<
+
+## install-dependencies : install most of the development dependencies via pip
 install-dependencies:
```

Comment:
Any issue with me changing this from `install-dependencies` to `install-dep`? It would make the help message more readable, but I don't know how many places in the documentation I would need to update this.